app/uk/gov/hmrc/eventhub/subscription/http/HttpResponseHandler.scala

```
@@ -48,26 +48,38 @@ class HttpResponseHandler(
                 logger.debug(s"pushed event: $event to: ${subscriber.uri}, marking as sent.")
                 subscriberEventRepository.sent(event).map(_ => resultF(Sent))
 
+              case StatusCodes.TooManyRequests =>
+                logger.warn(s"rate limit error: ${response.status} when pushing: $event to: ${subscriber.uri}.")
+                markAsFailed(event, resultF)
+
+              case StatusCodes.ServerError(_) =>
+                logger.warn(s"server error: ${response.status} when pushing: $event to: ${subscriber.uri}.")
+                markAsFailed(event, resultF)
+
               case StatusCodes.ClientError(_) =>
                 logger.warn(s"client error: ${response.status} when pushing: $event to: ${subscriber.uri}.")
```

I wonder if we should raise these as errors in kibana as opposed to warnings?